README.md

```diff
@@ -1,13 +1,17 @@
 # Falconieri
 
-Remote Provisioning Gateway
+Modern IP phones can contact a redirect service provider at boot time
+and discover their PBX address with it.
+
+The Falconieri project is a RPS (Redirect and Provisioning Service) gateway 
+that helps to store the phone provisioning URL in the phone vendor redirect service.
 
 Supported providers:
 
-* [**Snom**](https://service.snom.com/display/wiki/XML-RPC+API)
-* [**Gigaset**](https://teamwork.gigaset.com/gigawiki/display/GPPPO/Gigaset+Redirect+server)
-* **Fanvil**
-* [**Yealink**](http://support.yealink.com/documentFront/forwardToDocumentDetailPage?documentId=257)
+* [Snom](https://service.snom.com/display/wiki/XML-RPC+API)
+* [Gigaset](https://teamwork.gigaset.com/gigawiki/display/GPPPO/Gigaset+Redirect+server)
+* [Fanvil](https://fdps.fanvil.com/)
```

Comment:
The documentation in the fanvil link is only for the administration portal, not about the XML-RPC APIs.
And btw why have you removed the bold formatting? :)

Reply:
About the Fanvil link I agree it is not really good, but it's better than nothing IMO. At least it provides an idea of the subject...

About strong emphasis, it's a matter of style. IMO it is not required because
- it is not a title 
- there's no surrounding text
- the text itself is a link which is already kinda emphasis

Reply:
Ok, for the formatting, but I still think that the linked documentation adds no value in this context, all the other vendors' documentations are about the APIs of their RPSs and for Fanvil such public documentation simply don't exist.